.gitignore

```diff
@@ -119,3 +119,7 @@ warnings.txt
 *.js.map
 *.d.ts.map
 *.bak
+
+# Eng Tools
+eng/tools/TypeSpecValidation/dist
+!eng/tools/TypeSpecValidation/cmd/*.js
```

Comment:
Shouldn't this be covered by the *.js above?

Reply:
think we want the opposite here, those files need to be included.

Reply:
Oh I did miss the ! but I thought we were only committing the TS files.

Reply:
We are no longer committing any **generated** JS files.  But we need to commit one tiny "shim" JS file to redirect to the generated JS code.  This is required to make "Node CLI" tools work if they are compiled during "postinstall".  It also provides a cleaner separation between the TS code and the executable JS file (the shebang lives in the JS shim, not the TS code).

Reply:
This is the same pattern used for packages like `@typespec/compiler` itself.